python/spacewalk/spacewalk-backend.changes

```diff
@@ -1,3 +1,15 @@
+-------------------------------------------------------------------
```

Comment:
No need to have 2 distinct entries in the changelog. And no need to set the header for the entry, check with the other PRs for the formatting. For example: https://github.com/uyuni-project/uyuni/pull/6023/files

Reply:
ah okay, I followed some linked opensuse docs that said to use the output of `osc vc` so I used that, I'll try to update those too if I remember.

Reply:
The "authoritative" source for the changelogs would be https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs (openSUSE changelog guidelines still apply)